Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py

```diff
@@ -7,6 +7,7 @@
     ),
     outputCommands = cms.untracked.vstring(
         'keep *_ALCARECOMuAlCalIsolatedMu_*_*', 
+        'keep *_generalTracks_*_*',
```

Comment:
I am working on the RelVal stuff - I required some input from PPD conveners so it was delayed a few days. Assuming my current tests work I should have an answer by today/tomorrow conerning the event size. Yuriy is back in the office today too so I'll chat to him about the generalTracks questions.